### PR TITLE
version: Move `quick-links` CSS class into CSS module

### PR DESCRIPTION
**Pull request by Turbo87**
r? @locks

### CODE REVIEW

File: app/styles/crate/version.module.scss [hunk 1]
```diff
@@ -5,6 +5,26 @@
     top: 4px;
 }
 
+ul.quick-links {
```
Comment (locks) [inline app/styles/crate/version.module.scss]: do you need the `ul` specifier?

Comment (Turbo87) [reply app/styles/crate/version.module.scss]: not strictly necessary. do you want me to remove it?

Comment (Turbo87) [reply app/styles/crate/version.module.scss]: removed

**Pull request merged**